napbot/extensions/music/song.py

```diff
@@ -1,14 +1,26 @@
+import concurrent.futures
+from asyncio import TaskGroup, Task
 from collections import deque
 import io
 import itertools
 import asyncio
 import contextlib
 import re
 import os
-from typing import overload
+from ctypes.wintypes import HTASK
```

Comment:
lots of unused imports

Reply:
my bad pycharm loves importing random stuff, i'll run a clean